lib/main.dart

```diff
@@ -1,125 +1,145 @@
+                        Stack(
+                          children: [
+                            Image.network(m.backdrop),
+                            Positioned(
+                              bottom: 0,
+                              right: 20,
+                              child: Text(
+                                m.originalTitle,
+                                style: const TextStyle(
+                                  fontWeight: FontWeight.bold,
+                                  fontSize: 25,
+                                  color: Colors.white,
+                                ),
+                                textAlign: TextAlign.center,
+                              ),
+                            ),
+                          ],
+                        ),
+                        Padding(
+                          padding: const EdgeInsets.all(16.0),
+                          child: Row(
+                            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                            children: [
+                              MiddleText(s: 'Available on ${m.releaseDate}'),
+                              MiddleText(s: m.voteAverage),
+                            ],
+                          ),
+                        ),
+                        Row(
+                          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                          children: m.getGenreElements(),
+                        ),
+                        const SizedBox(
+                          height: 20,
+                        ),
+                        Stack(
+                          children: [
+                            Image.network(m.poster),
+                            const Positioned(
+                              top: 15,
+                              right: 15,
+                              child: Column(
+                                children: [
+                                  Icon(
+                                    Icons.favorite_outlined,
+                                    size: 55,
+                                    color: Colors.red,
+                                  ),
+                                  FloatingActionButton(
+                                    onPressed: null,
+                                    child: Icon(Icons.add),
+                                  ),
+                                ],
+                              ),
+                            ),
+                            Positioned(
+                              width: 350,
+                              top: 170,
+                              right: 10,
+                              child: Text(
+                                m.overview,
+                                style: const TextStyle(
+                                  fontWeight: FontWeight.w900,
+                                  fontSize: 20,
+                                  color: Colors.white70,
```

Comment:
You could create a separate file with all of your text styles to import, instead of having them spreaded throughout the code.

Reply:
done